src/olympia/lib/settings_base.py

```diff
@@ -926,11 +921,6 @@ def get_db_config(environ_var, atomic_requests=True):
             'js/impala/stats/table.js',
             'js/impala/stats/stats.js',
         ),
-        'zamboni/admin': (
-            'js/zamboni/admin.js',
-            'js/zamboni/admin_features.js',
-            'js/zamboni/admin_validation.js',
-        ),
```

Comment:
if this the last place these files are referenced (and I assume so) then we should remove the files in static too.  Or note this down in a follow-up issue (finding unused files later gets much more tedious.)

Reply:
dbd5610447